<launch>
  <include file="$(find realsense2_camera)/launch/rs_camera.launch">
    <arg name="align_depth" value="true" />
  </include>
  
  <node pkg="openpifpaf_ros" type="openpifpaf_ros_node.py" name="openpifpaf_ros">
    <param name="_in_topic" value="/camera/color/image_raw" />
    <param name="_out_topic" value="/human_pose" />
  </node>
  
  <node pkg="rgbd_pose_3d" type="rgbd_pose_3d_node.py" name="rgbd_pose_3d">
    <param name="_in_depth_topic" value="/camera/aligned_depth_to_color/image_raw" />
    <param name="_in_depthinfo_topic" value="/camera/aligned_depth_to_color/camera_info" />
    <param name="_in_pose_topic" value="/human_pose" />
    <param name="_out_topic" value="/human_pose_3d" />
  </node>
</launch>

